pennylane_forest/qpu.py

```diff
@@ -66,28 +61,19 @@ class QPUDevice(QVMDevice):
         readout_error (list): specifies the conditional probabilities [p(0|0), p(1|1)], where
             p(i|j) denotes the prob of reading out i having sampled j; can be set to `None` if no
             readout errors need to be simulated; can only be set for the QPU-as-a-QVM
-        symmetrize_readout (str): method to perform readout symmetrization, using exhaustive
+        symmetrize_readout (pyquil.experiment.SymmetrizationLevel): method to perform readout symmetrization, using exhaustive
             symmetrization by default
         calibrate_readout (str): method to perform calibration for readout error mitigation,
             normalizing by the expectation value in the +1-eigenstate of the observable by default
 
     Keyword args:
-        forest_url (str): the Forest URL server. Can also be set by
-            the environment variable ``FOREST_SERVER_URL``, or in the ``~/.qcs_config``
-            configuration file. Default value is ``"https://forest-server.qcs.rigetti.com"``.
-        qvm_url (str): the QVM server URL. Can also be set by the environment
-            variable ``QVM_URL``, or in the ``~/.forest_config`` configuration file.
-            Default value is ``"http://127.0.0.1:5000"``.
-        compiler_url (str): the compiler server URL. Can also be set by the environment
-            variable ``COMPILER_URL``, or in the ``~/.forest_config`` configuration file.
-            Default value is ``"http://127.0.0.1:6000"``.
-        timeout (int): number of seconds to wait for a response from the client.
         parametric_compilation (bool): a boolean value of whether or not to use parametric
```

Comment:
Should we add `compiler_timeout` and `execution_timeout`?

Reply:
Good catch.